compiler/rustc_codegen_cranelift/src/bin/cg_clif_build_sysroot.rs

```diff
@@ -8,7 +8,6 @@
 
 #![feature(rustc_private)]
 
-extern crate rustc_data_structures;
```

Comment:
Looks like you didn't update cg_clif to the 2021 edition in it's `Cargo.toml`.

Reply:
This change specifically comes from the move to SourceType::InTree in compile.rs for codegen backends. But yes, I didn't end up migrating cranelift -- I guess tidy ignores it -- I'll do that.